md/README.md

```diff
@@ -0,0 +1,90 @@
+# Mention Detection
+
+A mention detection module that annoatates all mentions of the given TextAnnoatation
+
+## Introduction
+
+### Mention Detection
+
```

Comment:
You can drop these two headers and combine this paragraph with the previous sentence.